term_posix.go

```diff
@@ -120,6 +120,20 @@ func (t *Term) SetRTS(v bool) error {
 	}
 }
 
+// SetHwFlowCtrl specifies whether hardware flow control is enabled
+func (t *Term) SetHwFlowCtrl(enabled bool) error {
```

Comment:
I'd like to see this presented as an option, see #13. 

I'd like to see a friendlier name, like FlowControl(term.Hardware|term.Software) also as an option, etc